dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -24,17 +25,31 @@ def login
     elsif params[:platform_id]
       query_params = {platform_id: params[:platform_id]}
     else
-      return unauthorized_status
+      return log_unauthorized(
+        "Missing required parameters for LTI authentication",
```

Comment:
Nit: Noticing we haven't been consistent with using single quotes for all non-interpolated strings. Some of these already existed (like all the `require` statements), but you have a mix of both single and double in this new logging code. Should we try to standardize on this new code using single quotes, and do a follow up PR that changes the rest of the doubles to singles?

Reply:
Oh good catch, I can definitely switch to single quotes. I think Copilot sometimes spits out double quotes even though I'm used to doing single quotes from my JS days.